packages/grafana-ui/src/components/Select/SelectBase.test.tsx

```diff
@@ -199,7 +199,7 @@ describe('SelectBase', () => {
       const selectEl = screen.getByLabelText('My select');
       expect(selectEl).toBeInTheDocument();
 
-      await selectOptionInTest(selectEl, 'Option 2');
+      await select(selectEl, 'Option 2', { container: document.body });
```

Comment:
**Question:** could we reference the moved `selectOptionInTest()` here as well, or would that add it to the bundle?

Maybe I am wrong, but probably it shouldn't be added to the bundle if it is only referenced in a test, right?

```javascript
import { selectOptionInTest } from 'test/helpers/selectOptionInTest';
```

Reply:
> could we reference the moved selectOptionInTest() here as well, or would that add it to the bundle?

Yes we could and it wouldn't be added to the bundle. It was being included because `plugin_loader` pulls in all of `grafana/ui` so plugins can access it and the `selectOptionInTest` function was exported from `grafana/ui` entry point causing webpack / rollup to bundle it.